### DISCUSSION TITLE
After using the eze plug-in, two identical duplicate files will appear.

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#928] [Q&A]
After using the eze plug-in, two identical duplicate files will appear.

**Comment by Difegue** [comment]
Hmm, maybe the metadata editing doesn't update the search indexes properly, leading to dupes in search with the same ID?  
Do they go away if you use "Reset Search cache" + "Rescan archive directory"?

**Reply by gittobabbuu** [reply]
Having the exact same problem with the Hitomi plugin.

This is not solved by "Reset Search cache" + "Rescan archive directory".

**Comment by inertia42** [comment]
Having the exact same problem with the E-hentai plugin.

**Reply by Difegue** [reply]
Does Reset Search cache fix it? I'm aware of an issue with search index generation where it can generate multiple entries for the same ID in certain cases(not sure which cases yet), so maybe it won't, but it's worth trying 🤔

**Reply by inertia42** [reply]
Thank you for your reply.
Resetting the search cache will not resolve the issue. However, when sorting by any field except the title, duplicate items will not appear.
The only way found to completely fix this issue  is by restarting the Docker container.

**Comment by inertia42** [comment]
> Thank you for your reply. Resetting the search cache will not resolve the issue. However, when sorting by any field except the title, duplicate items will not appear. The only way found to completely fix this issue is by restarting the Docker container.

I discovered a different way to solve the problem after trying several methods. Go to `Setting - Global Settings - Clean Database` and click it. This will remove the duplicate items.
